### ISSUE TITLE
Getting logits scores

### ISSUE THREAD

**Issue by Sakin101** [microsoft/LLaVA-Med#132]
model.generate() only return output_ids how do i get the logits scores
